### DISCUSSION TITLE
Rate and Duration of infusions

### DISCUSSION THREAD

**Discussion by discovererspirit** [nlmixr2/nlmixr2#78] [Q&A]
Hi everyone! 
This is my first post asking for some help, so if this is not the place let me know and I would appreciate your guidance.

I have a two compartment model for a monoclonal antibody from a paper that I would like to use for TDM.
I have my dataset simulated with concentrations from the analitycal ecuations in an Excel file.
If I load the datase and fit the model with **No** rate or duration on the dataset, the fit seems to be ok, but,
the monoclonal infusion antibody lasts 2 hours.
if I load the dataset with one dose of 350mg and the infusion duration/rate, the fit seems to be no longer good and goes always upwards?
I have tried different configurations of DUR (0.083days) / RATE (4200) configurations and couldn't find where is the mistake!? 
Any help would be very much appreciated ! :)
[2Cmt.txt](https://github.com/nlmixr2/nlmixr2/files/10014451/2Cmt.txt)
[ati_imm_13_etaVc_etaVp_etaCl.csv](https://github.com/nlmixr2/nlmixr2/files/10014470/ati_imm_13_etaVc_etaVp_etaCl.csv)

**Comment by mattfidler** [comment]
It seems this model is over-parameterized for the data you have.  That could be the reason why it is going to an odd location?

Also starting at a residual variability of 0 doesn't much make sense.

**Reply by mattfidler** [reply]
Note by adapting your code, simulating your model with one iiv, everything seems to work OK...

```r
library(ggplot2)
library(nlmixr2)
library(rxode2)
library(lattice)
library(tidyverse)
rm(list = ls())

set.seed(42)
rxSetSeed(42)

## Carga datos
#DATOS <- read.csv(file = "ati_imm_13_etaVc_etaVp_etaCl.csv", sep=';', header = T)

DATOS <- et(amt=350, dur=0.083) %>%
  et(seq(0.2, 24, by=0.5)) %>%
  et(id=1:12)

cov <- data.frame(id=1:12, WT=rnorm(12, 65, 5), ALB=rnorm(12, 4.1, 1), ati=round(runif(12)), imm=round(runif(12)))

DATOS <- merge(DATOS, cov)

# Transformación logarítmica
DATOS$LWT=    log(DATOS$WT)
DATOS$LW65=   log(DATOS$WT/65)
DATOS$LALB41= log(DATOS$ALB/4.1)

#### Modelo
DOS.cmt <- function() {
  ini({
    ltCl  <- log(5.42)   ; label("Ln del valor típico del Cl")    #mL/kg/day
    ltVc  <- log(52.4)   ; label("Ln del valor típico del  Vc")   #mL/kg
    ltQ   <- log(2.26)     # ln typical value of Aclaramiento intercompartimental #mL/kg/day
    ltVp  <- log(19.6)     # ln typical value of Volumen periférico
    lATI <- log(1.291)          # ln base ati
    lIMM <- log(0.863)          # ln base imm
   
    BETA_WT65_CL  <- fix(-0.313)  #allometric exponent WT  on cl
    BETA_ALB41_CL <- fix(-0.855)  #allometric exponent ALB on cl
    BETA_WT65_Vc   <- fix(-0.233) #allometric exponent WT  on Vc    
    BETA_WT65_Vp   <- fix(-0.588) #allometric exponent WT  on Vp    
    
    eta.Cl   ~ 0.307     ; label("eta_Cl")
    #eta.Vc   ~ 0.126     ; label("eta_Vc")
    #eta.Vp   ~ 0.553     ; label("eta_Vp")
    
    add.sd  <- 0.371 #0.371
    prop.sd <- 0.29 #0.292
  })
  model({
    Cl <- exp(ltCl + LWT + BETA_WT65_CL*LW65 +  BETA_ALB41_CL*LALB41 + ati*lATI+ imm*lIMM -3*log(10) + eta.Cl)
    Vc <- exp(ltVc + LWT + BETA_WT65_Vc*LW65 -3*log(10) ) #+ eta.Vc  )   # Liters
    Vp <- exp(ltVp + LWT + BETA_WT65_Vp*LW65 -3*log(10) ) #+eta.Vp 
    Q  <- exp(ltQ  + LWT -3*log(10) )

    linCmt() ~ add(add.sd) + prop(prop.sd) #concentration expected in mg/L equal to MICROg/miliL 
  })
}


dat <- nlmixr2(object=DOS.cmt, data=DATOS, est="rxSolve", control=rxControl(addDosing=TRUE)) %>%
  dplyr::rename(DV=sim)


fitF <- nlmixr2(object=DOS.cmt, data=dat, est="focei")

fitF
```

**Reply by discovererspirit** [reply]
First of all, thank you very much for your quick answer and the hints in the code !
I was kind of stuck and this gives me some ligth. But since  I'm a beginner and I think it's not exactly what I'm trying to do,  it's going to take me a little time, but I'm going to think about your suggestions, to do some test with the code and I'll come back! ;)

**Reply by mattfidler** [reply]
Good luck!
